inc/preview/namespace.php

```diff
@@ -27,7 +27,7 @@ function setup() : void {
  * @return bool True to show the preview selector, false otherwise.
  */
 function should_show_selector() : bool {
-	return is_admin_bar_showing() && ! is_admin() && current_user_can( 'edit_posts' );
+	return is_admin_bar_showing() && ! is_admin() && current_user_can( 'edit_posts' ) && count( Audiences\query_audiences() ) > 0;
```

Comment:
This solves the problem, but I wonder if it should have explanatory text instead so we're encouraging more use of audiences?

Reply:
Hmm. Could have a link to the audience editor admin page as well perhaps. I don't know really, do you want to update this with that suggestion? If you're not sure lets just merge this for now and create a new issue for your suggestion.